config/kubernetes/org.yaml

```diff
@@ -1105,21 +1105,24 @@ teams:
   cloud-provider-openstack-admins:
```

Comment:
Btw can you move the openstack related teams from here to `config/kubernetes/sig-openstack/teams.yaml`?

That way, oenstack leads can approve changes directly from the next time :)

Reply:
Done